cluster/gce/gci/configure-helper.sh

```diff
@@ -675,14 +684,19 @@ contexts:
     user: kube-apiserver
   name: webhook
 EOF
-    # This is the config for the image review admission controller.
-    cat <<EOF >/etc/admission_controller.config
-imagePolicy:
+GCP_IMAGE_REVIEW_CONFIG="imagePolicy:
```

Comment:
maybe insert a line break above and indent this-- it is inside the if block, right?

Reply:
I think "imagePolicy" is weirdly supposed to be at the same indentation level as the generic admission webook "kubeConfigFile"